modify setup instrcution for adjusting almond environment

diningyo opened the pull request:
When I tried to set up Chisel Bootcamp environment, jupyter-scala setup was changed:

- jupyter-scala github URL changed and redirecting to https://github.com/almond-sh/almond.git
- the module name was change to almond from jupyter-scala (maybe)
- almond setup method is changed to use coursier.

So, I changed README.md to match current setup instructions.

diningyo commented:
I realized making a mistake that almond setup instruction order.
So I modified setup instruction again in da5ea3a.

grebe commented:
Thank you for noticing this! I think it looks pretty good to me, I think we should have a couple people with different systems test it out. @chick @edwardcwang @azidar

edwardcwang commented:
I'll take a look.

diningyo commented:
Thank you for checking my pull request and modifying a mistake.

edwardcwang commented:
Sorry, I believe that I may have spoken too quickly. The install instructions work fine but since it defaults to Scala 2.12, structural types (used e.g. in IOs) is broken, so we need a way to stick a `-Xsource=2.11` flag to make it work.

(The error is also reproducible on Ubuntu 18.10)

diningyo commented:
Thanks your reporting that failed compilation. You are right and that problem is occurred on my environment:

- Ubuntu 16.04 LTS
- scala 2.12.7
- almond 0.1.9

Sorry for no checking to run chisel basic code, because I've just started to learn Chisel.
Anyway I'll check up your advice which add an `-Xsource=2.11` flags and try it.

diningyo commented:
In edwardcwang's advise, I changed Scala version to build almond in d4868c8.
The points are as follows:
- When cloning almond github, indicating `v0.1.9` version tag (for matching almond version obviously)
- The Scala version for building almond is changed from 2.12.7 to 2.11.12.

And I can run Chisel's code `io.out := io.in` in module2.

Could you check it again?

edwardcwang commented:
I'll take a look at it within the next few days. Was there not a way to get it to build with 2.12? For the Chisel builds we do with sbt we are able to use 2.12 with the `-Xsource=2.11` flag.

diningyo commented:
Thanks for replying my comment.

At first, I tried to use `-Xsource=2.11` flag, so I edited almond's build.sbt file and rebuilt it.
Added option is below.

```scala
scalacOptions ++= Seq(
  "-Xsource=2.11",
)
```
But it didn't work for me.
Is that trial wrong ?

After that trial, I checked .travis.yml file and found `scala: 2.11.12` at `include` section.
So I changed `SCALA_VERSION` setting. And this time, it works for me.
